silveroak-opentitan/aes/Arrow/_CoqProject

```diff
@@ -18,7 +18,6 @@ unrolled_naive_cipher.v
 unrolled_opentitan_cipher.v
 cipher_round.v
 key_expand.v
-cipher_control.v
```

Comment:
Is this supposed to be removed?

Reply:
Yeah, type unification is particularly flakey in the combination of this definition and the custom notation it uses. I will introduce a better fix for this in upcoming notational changes